datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -462,6 +523,28 @@ macro_rules! get_null_count_values {
     }};
 }
 
+/// Convert parquet column schema to arrow field.
+/// copy from arrow-rs
+/// TODO: consolidate code with arrow-rs
+/// TODO: change this API public in the arrow-rs
+/// crate::schema::parquet_to_arrow_field
+fn parquet_to_arrow_field(parquet_column: &ColumnDescriptor) -> Option<DataType> {
+    let type_ptr = parquet_column.self_type_ptr();
+    match type_ptr.get_basic_info().logical_type() {
+        // just handle the decimal type
```

Comment:
In this case, I just handle the decimal case, and don't change the logic for other data types.
Maybe there are bugs for other data types, I will add tests to check them.

Reply:
Maybe we could rename the function to `parquet_to_arrow_decimal_field` or something to make it more clear that this only applies to decimal types

Reply:
change the function name to `parquet_to_arrow_decimal_type`